packages/walletconnect-signing-manager/src/lib/walletconnect-signing-manager.ts

```diff
@@ -0,0 +1,163 @@
+import type { InjectedAccountWithMeta } from '@polkadot/extension-inject/types';
+import type { SigningManager } from '@polymeshassociation/signing-manager-types';
+
+import type { CreateParams, UnsubCallback } from '../types';
+import { changeAddressFormat, mapAccounts } from '../utils';
+import { WalletConnect } from './walletconnect';
+import { WalletConnectSigner } from './walletconnect/signer';
+
+export class WalletConnectSigningManager implements SigningManager {
+  private _ss58Format?: number;
+  private _genesisHash?: string;
+
+  private constructor(private readonly walletConnect: WalletConnect) {}
+
+  /**
+   * Create a Signing Manager that connects to a browser extension
+   *
+   * @param args.config - walletConnect configuration parameters
+   * @param args.appName - name of the dApp attempting to connect to the extension
+   * @param args.ss58Format - SS58 format for the extension in which the returned addresses will be encoded(optional)
+   * @param args.genesisHash - genesis hash of the target chain(optional)
```

Comment:
```suggestion
   * @param args.genesisHash - genesis hash of the target chain (optional)
```
(nit) I think the space makes it more readable